x-pack/plugins/lens/public/xy_visualization/visualization.test.ts

```diff
@@ -371,4 +371,142 @@ describe('xy_visualization', () => {
       expect(ops.filter(filterOperations).map((x) => x.dataType)).toEqual(['number']);
     });
   });
+
+  describe('#getErrorMessage', () => {
+    it("should not return an error when there's only one dimension", () => {
+      expect(
+        xyVisualization.getErrorMessage(
+          {
+            ...exampleState(),
+            layers: [
+              {
+                layerId: 'first',
+                seriesType: 'area',
+                xAccessor: 'a',
+                accessors: [],
+              },
+            ],
+          },
+          createMockFramePublicAPI()
+        )
+      ).not.toBeDefined();
+    });
```

Comment:
I know you were talking about this with @timroes, but I would like to get a summary of the decision about this one. I can imagine several ways of solving it, including putting it off for another PR.

Reply:
I had an idea about, maybe it can be implemented in here. @timroes opened the #81789 issue about meanwhile.